lib/src/modules/inventoryItem/InventoryItem.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import cx from 'classnames';
+import { bool, string } from 'prop-types';
+import { ButtonIconContained, TooltipWrapper } from 'lib';
+
+export function InventoryItem({
+  isDragging,
+  isDragPreview,
+  iconName,
+  className,
+  tooltipClassName,
+  tooltipText,
+  ...rest
+}) {
+  const classNames = cx(`inventory-item ${className}`, {
+    'inventory-item-dragging': isDragging,
+    'inventory-item-preview': isDragPreview
+  });
+
+  if (isDragging) {
+    return <div className={classNames} />;
+  }
+
+  return (
+    <TooltipWrapper
```

Comment:
🔴  Is there a reason why `TooltipWrapper` isn't composed outside of this?

This module could just focus on rendering an `InventoryItem` module, instead of worrying about every prop required for tooltips. Tooltips are so widely used (and easily composable) that I think in the long run it'd be much better if a lot of our component doesn't have props like `tooltipText`.

Reply:
I get what you mean, but I think we should leave it in, everywhere we're going to use this will use a tooltip. However, you absolutely don't have to pass `tooltipText` so you can pretend it doesn't exist. I'm also pretty sure that if you don't pass `tooltipText` to a tooltip it'll just render its children and none of the tooltip stuff!